### ISSUE TITLE
[Bug Report][2.6.12] V-Autocomplete should generate listbox adjacent to input instead of end of document

### ISSUE THREAD

**Issue by sidnc86** [vuetifyjs/vuetify#16112]
### Environment
**Vuetify Version:** 2.6.12
**Vue Version:** 2.6.11
**Browsers:** Chrome 107.0.0.0
**OS:** Windows 10

### Steps to reproduce
1. Either type inside the field, click the down arrow icon or press keyboard down arrow.
2. Let the listbox appear with foo, bar items.
3. Inspect the expanded listbox.
4. The listbox doesn't appear adjacent to input element.

### Expected Behavior
Listbox is expected to appear adjacent to input field and not at end of document when inspected from elements panel. This will also ensure that swiping gesture of screen reader like Voice Over on IPad/IOS, Talkback on Android, etc. can access the listbox immediately after the input field.

### Actual Behavior
The listbox attaches to end of document and hence on swiping, screen reader takes the Accessibility Focus to next field insted taking it to listbox when swiped from input field.

### Reproduction Link
[https://codepen.io/sidnc86/pen/PoaJrNG?editors=101](https://codepen.io/sidnc86/pen/PoaJrNG?editors=101)

### Other comments
From Accessibility standpoint, especially on touch screen devices with screen reader, using swipe gestures, users cannot find the listbox adjacent to input field. This works only on Android platform because of the aria-owns property. It should also be noted that this dropdown/combobox needs to comply with ARIA 1.2 spec which is the currently valid spec. This dropdown/combobox is complying with ARIA 1.1 spec which is now obsolete.
ARIA 1.2 expects following:
1. role="combobox", aria-expanded, aria-controls on input and not on wrapping div
2. It does not need aria-owns.
3. The down arrow icon should have accessible label but need not receive focus with tabindex="-1". This ensures that on touch-screen environments, screen reader users can expand the dropdown. And on keyboard environments where down arrow is available, this icon can be skipped from focusing.
